.travis.yml

```diff
@@ -173,7 +173,7 @@ jobs:
     #--- Build and deploy Debian packages ---
```

Comment:
We should review this comments: Build and deploy Debian packages, since we are not deploying anymore. ;)

Reply:
Fixed :-)